operations/cli_integration_test.go

```diff
@@ -1,4 +1,4 @@
-package cli
+package operations
```

Comment:
compile error:
operations/cli_integration_test.go:134:12: cannot use []string literal (type []string) as type string in field value
operations/cli_integration_test.go:135:4: cannot use false (type bool) as type []string in field value
operations/cli_integration_test.go:135:4: too few values in struct initializer
operations/cli_integration_test.go:346:14: cannot use []string literal (type []string) as type string in field value
operations/cli_integration_test.go:347:6: cannot use false (type bool) as type []string in field value
operations/cli_integration_test.go:347:6: too few values in struct initializer
operations/cli_integration_test.go:401:14: cannot use []string literal (type []string) as type string in field value
operations/cli_integration_test.go:402:6: cannot use false (type bool) as type []string in field value
operations/cli_integration_test.go:402:6: too few values in struct initializer
operations/cli_integration_test.go:414:14: cannot use []string literal (type []string) as type string in field value
operations/cli_integration_test.go:414:14: too many errors